iothub/device/tests/Transport/Mqtt/MqttIotHubAdapterTest.cs

```diff
@@ -111,7 +111,7 @@ public void TestAuthenticationChain()
             var passwordProvider = new Mock<IAuthorizationProvider>();
             var mqttIotHubEventHandler = new Mock<IMqttIotHubEventHandler>();
             var productInfo = new ProductInfo();
-            var options = new IotHubClientOptions();
+            var options = new IotHubClientOptions(new IotHubClientMqttSettings());
```

Comment:
Oops, these tests are supposed to be for MQTT, but without being explicit they were testing AMQP before. Fixing that, even though the default is now MQTT.